### DISCUSSION TITLE
GetLost map URL

### DISCUSSION THREAD

**Discussion by gpzninja** [osmandapp/OsmAnd#13030] [Maps Related]
Hi Guys, the default URL when installing GetLost online map is incorrect and the maps does not load.

GetLost provided me the correct URL and works well: 

**https:/live.getlost.com.au/{z}/{x}/{y}.jpg**

Hope someone can update this so that people can use it? GetLost have a nice Covid restriction boundary

**Comment by xmd5a2** [comment]
Fixed

**Reply by gpzninja** [reply]
Thank you !!
